Ginger/GingerCoreNET/RosLynLib/CodeProcessor.cs

```diff
@@ -44,10 +45,39 @@ public object EvalExpression(string expression)
             return task.Result;
         }
 
-        
-        public bool EvalCondition(string condition)
+        public static string GetResult(string Expression)
+        {
+            Pattern =   new Regex("{CS(\\s)*Exp(\\s)*=(\\s)*[^}]*}");
```

Comment:
I thought we will avoid regex because it is expensive, please explain

Reply:
As discussed there is no other option